index.html

```diff
@@ -5,6 +5,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Hamayoun</title>
 	<link rel="stylesheet" type="text/css" href="assets/css/style.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<script src="assets/js/script.js"></script>
 	<style>
 
 		@font-face {
```

Comment:
- [ ] <b>OPTIONAL:</b> It is not best practice to use internal CSS. It would nice to consider moving them to external file.